### ISSUE TITLE
kubectl command in readme invalid

### ISSUE THREAD

**Issue by drshrey** [agonzalezro/k8s-py-example#1]
```
➜  k8s-py-example git:(master) ✗ kubectl -f rc-0.1.yml service.yml --validate=false
Error: unknown command "service.yml" for "kubectl"
Run 'kubectl --help' for usage.
```

I think it should be  `kubectl create -f rc-0.1.yml service.yml --validate=false`

**Comment by agonzalezro** [comment]
Yep @drshrey, you are right. Also, I think you will need another `-f` after `rc-0.1yml`.

In other order of things, I doubt that you need `--validate=false` now, that was an old bug in Kubernetes.

If you want to create a PR with the change I will happily merge it. Thanks for the heads up!
